### PR TITLE
test(Hero): add e2e test for Hero component

### LINKED ISSUE #414: e2e testing: searching Repos
Accessibility should be tested. Also, some simple functionality.

Screen Shot 2022-11-29 at 9 36 11 PM

### PR DESCRIPTION
**Pull request by NsdHSO**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [x] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Fixes #414 

## Added tests?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by NsdHSO** [comment]
I don't know why CI is failing, because on localhost work good 
image

File: src/e2e-tests/hero.spec.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    "body:has-text(\"OpenSaucedStar us on GitHubSign inFind Open-Source Repositoriesto contribute tod\")",
+  )
+    .press("Meta+k");
+  await page.getByPlaceholder("Search repositories")
+    .fill("test");
+  const [page4] = await Promise.all([
+    page.waitForEvent("popup"),
+    page.getByRole(
+      "link",
+      { name: "goldbergyoni/javascript-testing-best-practices goldbergyoni/javascript-testing-best-practices 📗🌐 🚢 Comprehensive and exhaustive JavaScript & Node.js testing best practices (December 2022) DouglasMV NoriSte Userbit js-kyle mel-mouk 49 19.9k" },
```
Comment (0-vortex) [inline src/e2e-tests/hero.spec.ts:18]: can we have the testing repos be our owned repositories in https://github.com/orgs/open-sauced/repositories ?

**Review by 0-vortex — CHANGES_REQUESTED** [review]
LGTM! 👍 

Couple of minor typos to fix and this can get merged 🍕

File: src/e2e-tests/hero.spec.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import { test } from "@playwright/test";
+
+test("test2", async ({ page }) => {
+  await page.goto("https://hot.opensauced.pizza/discussed");
```
[the remark this answers is not in the document]
Comment (NsdHSO) [reply src/e2e-tests/hero.spec.ts]: Ok, tnx for PR.

File: src/e2e-tests/hero.spec.ts (hunk 1 above)
Comment (NsdHSO) [reply src/e2e-tests/hero.spec.ts]: Yes, I can. 
Tnx.

**Review by 0-vortex — CHANGES_REQUESTED** [review]
@NsdHSO you really need to adjust the URL to be a moving target. This PR is not going to get merged only checking absolute URLs. Resolving the conversation without making the change is not the way to go.

If you do not understand something please ask questions.

**Comment by NsdHSO** [comment]
@0-vortex I think the scope of this issue, is only to check the accessibility, but no worries I can parse all your request. Please can you be more clear about what means absolute URLs? These? `
{
  "/popular": "popular",
 "/upvoted": "upvoted",
  "/discussed": "discussed",
  "/recent": "recent",
  "/myVotes": "myVotes",
}
`

**Comment by NsdHSO** [comment]
Hi please @0-vortex  can you provide me more info about your comment?

File: src/e2e-tests/hero.spec.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import {
+  test,
+  expect,
+} from "@playwright/test";
+
+test("should test accessibility", async ({ page }) => {
+  await page.goto("https://hot.opensauced.pizza/");
```
Comment (0-vortex) [inline src/e2e-tests/hero.spec.ts:7]:
```suggestion
  await page.goto("/");
```

File: src/e2e-tests/hero.spec.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    page.getByRole(
+      "link",
+      { name: "open-sauced/hot open-sauced/hot 🍕The site that recommends the hottest projects on GitHub. Deadreyo ddsuhaimi NsdHSO hokagedemehin bdougie 45 207" },
+    )
+      .click(),
+  ]);
+
+  await page2.getByText("7d")
+    .click();
+  await page.locator("div:has-text(\"Find Open-Source Repositoriesto contribute today\")")
```
Comment (0-vortex) [inline src/e2e-tests/hero.spec.ts:56]: this looks like a typo but code is good now

**Pull request closed without merging**